renovate.json

```diff
@@ -25,10 +25,6 @@
       "groupName": "Emotion",
       "matchPackagePatterns": "@emotion/*"
     },
-    {
-      "groupName": "Font awesome SVG icons",
-      "matchPackagePatterns": "@fortawesome/*"
-    },
```

Comment:
Was this intentionally removed? I can't see it in the PR description.

Reply:
Yes, we don't have this dependency anymore. I'll update the description.